addons/paint-skew/addon.json

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Skewing in costume editor",
+  "description": "Hold Ctrl while scaling an object on its sides to skew it. Hold Alt while skewing an object to skew it from its center.",
+  "info": [
+    {
+      "text": "On macOS, use the Cmd key instead of the Ctrl key.",
```

```suggestion
      "text": "On macOS, use the Option key instead of the Alt key and Command key instead of the Control key.",
```

---

I'd say those 2 should be swapped:
```suggestion
      "text": "On macOS, use the Command key instead of the Control key and the Option key instead of the Alt key.",
```